### ISSUE TITLE
‘About the data’ and ‘Accessibility’ pages should be in a column two-thirds.

### ISSUE THREAD

**Issue by bsaadat** [PublicHealthEngland/coronavirus-dashboard#88]
The 'About the data' and 'accessibility' pages are slightly narrower than the main page - they should be consistent with the main page.

**Comment by bsaadat** [comment]
![image](https://user-images.githubusercontent.com/27902577/80505244-fbf70f80-896b-11ea-8b71-b6d6113b7b7c.png)

**Comment by bsaadat** [comment]
![image](https://user-images.githubusercontent.com/27902577/80505382-2648cd00-896c-11ea-8b0b-dd6dd7654c56.png)

**Comment by bsaadat** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [‘About the data’ and ‘Accessibility’ pages should be in a column two-thirds.](https://trello.com/c/7Q63mwRS/193-about-the-data-and-accessibility-pages-should-be-in-a-column-two-thirds)

**Comment by stuartmccoll** [comment]
Are you accepting contributions for issues? If so, I'd happily pick this up.

**Comment by Ha-san-ali** [comment]
Indeed we are @stuartmccoll 
This issue is related to this for context https://github.com/PublicHealthEngland/coronavirus-dashboard/pull/107

**Comment by stuartmccoll** [comment]
Hi, apologies - haven't had much time to look at this. I did initially pull the repository down and run it locally, where I wasn't able to recreate the issue - the widths were all aligned in my local build.
